Misc/NEWS.d/3.5.3rc1.rst

```diff
@@ -1984,7 +1984,7 @@ Haubenwallner.
 .. nonce: CLz6qy
 .. section: Build
 
-Rename --with-optimiations to --enable-optimizations.
```

Comment:
Did you check the Git log to see if this renaming was an actual configure flag spelling bugfix? Ditto for the 3.6 and 3.7 NEWS rst files.

Reply:
I'm wondering if it makes sense at all to go back through old versions and correct typos.

Reply:
Yeah. There is a risk of _introducing_ documentation errors and inconsistencies with such changes.

Reply:
We can start with the second question here from @willingc. That's my question to you. I included the changes in history files to showcase but I don't know if it should be included.

Reply:
@erlend-aasland I think https://github.com/python/cpython/pull/125569#pullrequestreview-2371548312 is how we should approach typo PRs.

Reply:
FWIW the spelling fix is appropriate. This is the original commit in 3.5: https://hg.python.org/cpython/rev/c0ea81315fb6

Reply:
> FWIW the spelling fix is appropriate. This is the original commit in 3.5: https://hg.python.org/cpython/rev/c0ea81315fb6

I know, and for such cases it is helpful to include this Git archeology in the description of the PR, for the convenience of the reviewer.